source/includes/examples/tf-example-main-devtest.rst

```diff
@@ -28,6 +28,14 @@
          region_name   = var.atlas_region
        }
      }
+     tags {
```

Comment:
the link in the PR description is incorrect because it's for cluster (which will be deprecated), the correct documentation for advanced_cluster is: https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster

so tags should be defined as: 
```	
tags {
  key   = "BU"
  value = "ConsumerProducts"
}
tags {
  key   = "TeamName"
TeamA}
etc.
```


same for example below